improve error message if file not found

If the `MODELICAPATH` is not set, then the error message below appears. This should be more informative than 
> Error: Modelica file 'undefined' does not exist.

It should instruct the user with a message like
```
error:   Modelica file '[add file name]' not found. You may need to set the MODELICAPATH environment variable
```

To reproduce, run
```
unset MODELICAPATH
node app.js \
	--log warn \
	-f Buildings/Controls/OBC/ASHRAE/G36_PR1/AHUs/MultiZone/VAV/Economizers/Controller.mo \
	-o html
```
The full message is currently:
```
error:   Modelica file 'undefined' does not exist.
/Users/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-json/lib/modelicaToJSON.js:24
    throw new Error(msg)
    ^

Error: Modelica file 'undefined' does not exist.
    at Object.toJSON (/Users/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-json/lib/modelicaToJSON.js:24:11)
    at getJsons (/Users/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-json/lib/parser.js:166:26)
    at Object.getJSON (/Users/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-json/lib/parser.js:146:16)
    at Object.<anonymous> (/Users/mwetter/proj/ldrd/bie/modeling/github/lbl-srg/modelica-json/app.js:76:15)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:394:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:509:3
make: *** [run] Error 1
```

--- a/lib/modelicaToJSON.js
+++ b/lib/modelicaToJSON.js
@@ -8,9 +8,10 @@ var logger = require('winston')
   *
   *@param {String} modelicaFile full path of modelica file to be parsed.
   *@param {String} MODELICAPATH system variable MODELICAPATH
+  *@param {String} parseMode parsing mode
   *@return A JSON representation of the Modelica file.
   */
-function toJSON (modelicaFile, MODELICAPATH) {
+function toJSON (modelicaFile, parseMode, MODELICAPATH) {
   const parser = pa.join(__dirname, '..', 'java', 'moParser.jar')
 
   if (!fs.existsSync(parser)) {
@@ -19,7 +20,12 @@ function toJSON (modelicaFile, MODELICAPATH) {
     throw new Error(msg)
   }
   if (!fs.existsSync(modelicaFile)) {
-    const msg = "Modelica file '" + modelicaFile + "' not found. You may need to set the MODELICAPATH environment variable."
+    var msg
+    if (parseMode === 'cdl') {
+      msg = "Modelica file '" + modelicaFile + "' not found. You may need to set the MODELICAPATH environment variable."
+    } else {
+      msg = "Modelica package '" + modelicaFile + "' not found. You may need to set the MODELICAPATH environment variable."
+    }
     logger.error(msg)
     throw new Error(msg)
   }

--- a/lib/modelicaToJSON.js
+++ b/lib/modelicaToJSON.js
@@ -19,7 +19,7 @@ function toJSON (modelicaFile, MODELICAPATH) {
     throw new Error(msg)
   }
   if (!fs.existsSync(modelicaFile)) {
-    const msg = "Modelica file '" + modelicaFile + "' does not exist."
+    const msg = "Modelica file '" + modelicaFile + "' not found. You may need to set the MODELICAPATH environment variable."
     logger.error(msg)
     throw new Error(msg)
   }

--- a/lib/parser.js
+++ b/lib/parser.js
@@ -92,7 +92,7 @@ function getNewData (topLevelWithin, jsonData, data, declaringModelicaFileName, 
     // However, these will be in the same modelica file. Hence, we check if
     // we parsed this file already.
     if (fileName) {
-      const jsonContent = mj.toJSON(fileName, MODELICAPATH)
+      const jsonContent = mj.toJSON(fileName, parseMode, MODELICAPATH)
       return jq.simplifyModelicaJSON(jsonContent, parseMode)
     } else { // fileName is undefined
       logger.warn('Did not find Modelica file for class ' +
@@ -163,7 +163,7 @@ function getJsons (moFile, parseMode, outputFormat) {
   logger.debug('Entered parser.getJSON for ' + moFile)
   const MODELICAPATH = process.env.MODELICAPATH
 
-  const jsonContent = mj.toJSON(moFile, MODELICAPATH)
+  const jsonContent = mj.toJSON(moFile, parseMode, MODELICAPATH)
   const topLevelWithin = jsonContent.within ? jsonContent.within[0] : null
 
   var data = []

--- a/lib/util.js
+++ b/lib/util.js
@@ -181,12 +181,16 @@ function getMoFiles (parseMode, files) {
     if (fse.existsSync(files)) {
       moFiles.push(files)
     } else {
+      var tempPath
       for (var j = 0; j < MODELICAPATH.length; j++) {
-        var tempPath = path.join(MODELICAPATH[j], files)
+        tempPath = path.join(MODELICAPATH[j], files)
         if (fse.existsSync(tempPath)) {
           moFiles.push(tempPath)
           break
         }
+      }
+      if (!fse.existsSync(tempPath)) {
+        moFiles.push(files)
       }
     }
   }

--- a/lib/util.js
+++ b/lib/util.js
@@ -170,10 +170,14 @@ function getMoFiles (parseMode, files) {
         if (fse.existsSync(tempFolder)) break
       }
     }
-    var fileList = getFiles(tempFolder)
-    moFiles = fileList.filter(function (obj) {
-      if (obj.endsWith('.mo')) { return true } else { return false }
-    })
+    if (!fse.existsSync(tempFolder)) {
+      moFiles.push(files)
+    } else {
+      var fileList = getFiles(tempFolder)
+      moFiles = fileList.filter(function (obj) {
+        if (obj.endsWith('.mo')) { return true } else { return false }
+      })
+    }
   } else {
     if (!files.includes('.mo')) {
       throw new Error('Wrong file input, it should be a single moFile.')

